dhis-2/dhis-services/dhis-service-administration/src/main/java/org/hisp/dhis/apphub/DefaultAppHubService.java

```diff
@@ -76,6 +80,28 @@ public DefaultAppHubService( RestTemplate restTemplate, AppManager appManager,
         this.dhisConfigurationProvider = dhisConfigurationProvider;
     }
 
+    @Override
+    public String getAppHubApiResponse( String apiVersion, String query )
+    {
+        validateQuery( apiVersion );
+        validateQuery( query );
+
+        apiVersion = sanitizeQuery( apiVersion );
+        query = sanitizeQuery( query );
+
+        String appHubApiUrl = dhisConfigurationProvider.getProperty( ConfigurationKey.APPHUB_API_URL );
+
+        log.info( "Get App Hub response, base URL: '{}', API version: '{}', query: '{}'", query, apiVersion, appHubApiUrl );
+
+        String url = String.format( "%s/%s/%s", appHubApiUrl, apiVersion, query );
+
+        log.info( "App Hub request URL: '{}'", url );
+
+        ResponseEntity<String> response = restTemplate.exchange( url, HttpMethod.GET, getJsonRequestEntity(), String.class );
```

Comment:
This does not forward on any parameters to the apps.dhis2.org API - for example, the following queries all print `App Hub request URL: 'https://apps.dhis2.org/api/v1/apps'` and return the full list of 39 apps without any filtering applied:

http://d2.winged.tech:8080/api/appHub/v1/apps?channel=canary (should return 0 apps)
http://d2.winged.tech:8080/api/appHub/v1/apps?dhis2_version=2.22 (should return 8 apps)
http://d2.winged.tech:8080/api/appHub/v1/apps?dhis2_version=2.34 (should return 27 apps)

Reply:
Including the query string now (see `ContextUtils`).